src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -592,17 +594,17 @@ public SecurityIdentifier Owner
 
                 if (_owner == null)
                 {
-                    using (SafeLocalAllocHandle tokenOwner = GetTokenInformation(_safeTokenHandle, TokenInformationClass.TokenOwner))
+                    using (SafeLocalAllocHandle? tokenOwner = GetTokenInformation(_safeTokenHandle, TokenInformationClass.TokenOwner))
                     {
-                        _owner = new SecurityIdentifier(tokenOwner.Read<IntPtr>(0));
+                        _owner = new SecurityIdentifier(tokenOwner!.Read<IntPtr>(0));
```

Comment:
The call to GetTokenInformation won't return null because true isn't being passed to the last optional argument.  It'd be better to just `!` on the GetTokenInformation call, and remove the `?` on `SafeLocalAllocHandle` and the `!` on `tokenOwner`.

Reply:
Good call. I think I might explicitly pass that optional argument for better emphasis.